### ISSUE TITLE
Missing focus styles for focusable elements in Forem administration pages

### ISSUE THREAD

**Issue by nickytonline** [forem/forem#10057]
**Describe the bug**

**To Reproduce**

Tab through fields in the admin UI on any page. Elements that receive focus do not show any visual cues that it has been focused on.

**Expected behavior**

A visual cue when a focusable element receives focus.

**Screenshots**

![image](https://user-images.githubusercontent.com/833231/91592971-565da800-e92d-11ea-868e-871a6f02d425.png)

**Desktop (please complete the following information):**

N/A

**Smartphone (please complete the following information):**

N/A

**Additional context**

**Comment by nickytonline** [comment]
This will required input from product design. cc: @lisasy, @ludwiczakpawel.

**Comment by ludwiczakpawel** [comment]
That's definitely coming! In fact, we're couple days aways from pushing some bigger visual changes to Admin side. These will NOT address the issue you mentioned, but that's definitely
gonna be the next step. So if everything goes smooth, expect fix for this particular issues at the end of next week 🙌

**Comment by nickytonline** [comment]
@ludwiczakpawel, just wondering if this is still something that you'll be working on or can a contributor work on this? If so, please label it with the Hacktoberfest label. Thanks!

**Comment by ludwiczakpawel** [comment]
Labeled!

**Comment by kldickenson** [comment]
Is this issue ready for a Hacktoberfest contributor? I have a background in accessibility and I would like an opportunity to help out.
-- Karen

**Comment by nickytonline** [comment]
@kldickenson, it is. Assigning it to you.

**Comment by nickytonline** [comment]
@kldickenson, to get access to the admin section, you will need to do the following from a terminal window/

- run `rails console` to open the Ruby on Rails  console.
- from the Rails console prompt, run the following

```
u = User.last;
roles = ["single_resource_admin", "super_admin", "trusted", "tech_admin", "config"];
roles.each { |r| u.add_role(r) };
u.add_role :single_resource_admin, Config; ```

This will grant you all the privileges you need to work on the admin section of the application.

**Comment by kldickenson** [comment]
@nickytonline
I was finally able to get my local docker forem/forem up and running and also figured out how to get into the rails console ("docker-compose exec rails rails console"). I ran the rails console commands you sent. 
Was it supposed to automatically log me in or are there special credentials (email/pw) I should use? 
Was there anything I should have replaced with my own info in the commands?
I haven't work in ruby on rails before so I'm going to need a little extra guidance.
Thanks ahead of time,
Karen

**Comment by nickytonline** [comment]
@kldickenson, so once you're local Forem is up and running, log in with your Twitter or GitHub. Once you're logged in, run those commands in the rails console, and then your user will be an admin and have access to those pages. Does that clarify things?

**Comment by kldickenson** [comment]
@nickytonline  Bummer, does not seem to be working in Docker. It will not let me create a new user with GitHub or Twitter login (I get **404** for GitHub and **oAuth::Unauthorized** for Twitter). I even tried User.create inside the rails console but was not successful. Gitpod will not let me do this either.
I so wanted to help but cannot seem to into the admin side.
Any more ideas to get in? If not, time to unassign me.
-- Karen

**Comment by nickytonline** [comment]
@kldickenson, did you set up your environment variables for Twitter and GitHub? You need to create OAuth apps for Twitter and GitHub to be able to log on with them. See https://docs.forem.com/backend/authentication/.

**Comment by kldickenson** [comment]
@nickytonline  Tried with GitHub, no luck. Working on Twitter now.

**Comment by kldickenson** [comment]
@nickytonline No luck with Twitter either. Twitter has changed its process for creating a developer account and create Apps. It no longer has an option to "Sign in with Twitter". It now offers "3-legged OAuth" and provides you a Twitter API key, a Twitter API secret key, and a Bearer token.
I have learned a lot getting my local dev environment up and running but I think I have reached my threshold. It's time to bow out and let someone else give this a try.
-- Karen

**Comment by nickytonline** [comment]
OK, I'll unassign you. If there are points in the documentation that we need to improve or revise to get future contributors up and running let me know and we can work on improving the documentation.

**Comment by mikeydorje** [comment]
Code review suggests this may be resolved or no longer applicable due to changes in this area since the issue was filed. Closing due to age and low activity. Please reopen with current repro steps if this persists.
